stats/perf-stat

```diff
@@ -121,6 +102,31 @@ def output_interval(prev_time, time)
   $interval_metrics_val.clear unless $interval_metrics_val.empty?
 end
 
+def update_interval(prev_time, time)
+  interval = time - prev_time
+  puts "time: #{time}"
+  $interval_results_sum.each do |key, value|
+    puts "i.#{key}: #{value / interval}"
+  end
+  calc_addon_keys($interval_results_sum, $interval_results_nr,
+                  interval, 1, 'i.')
+end
+
+def ignore_result?(prev_time, time)
+  if time <= $stats_begin_time
```

Comment:
thanks, based on this code, prefer here to have an early return
```
return true if time <= $stats_begin_time
```

Reply:
Fixed, thanks!